src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -2,11 +2,16 @@
 
 public class BinaryString {
 
-    /**
-     * Напишите метод toBinaryString(), который принимает на вход целое число value,
-     * а возвращает String с представлением этого числа в двоичном виде.
-     */
     public String toBinaryString(int value) {
-        return null;
+        if (value == 0) {
+            return "0";
+        }
+        StringBuilder itog = new StringBuilder();
```

Comment:
```suggestion
        StringBuilder result = new StringBuilder();
```
let's write in English :)

Reply:
)))))